plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/binance/v4/BinanceBeanV4.java

```diff
@@ -0,0 +1,314 @@
+package io.everytrade.server.plugin.impl.everytrade.parser.exchange.binance.v4;
+
+import com.univocity.parsers.annotations.Format;
+import com.univocity.parsers.annotations.Headers;
+import com.univocity.parsers.annotations.Parsed;
+import io.everytrade.server.model.Currency;
+import io.everytrade.server.model.TransactionType;
+import io.everytrade.server.plugin.api.parser.BuySellImportedTransactionBean;
+import io.everytrade.server.plugin.api.parser.DepositWithdrawalImportedTransaction;
+import io.everytrade.server.plugin.api.parser.FeeRebateImportedTransactionBean;
+import io.everytrade.server.plugin.api.parser.ImportedTransactionBean;
+import io.everytrade.server.plugin.api.parser.TransactionCluster;
+import io.everytrade.server.plugin.impl.everytrade.parser.ParserUtils;
+import io.everytrade.server.plugin.impl.everytrade.parser.exception.DataIgnoredException;
+import io.everytrade.server.plugin.impl.everytrade.parser.exchange.ExchangeBean;
+import lombok.NonNull;
+
+import java.math.BigDecimal;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+
+@Headers(sequence = {"Operation", "UTC_Time", "Account", "Coin", "Change", "Remark", "User_ID"}, extract = true)
+public class BinanceBeanV4 extends ExchangeBean {
+
+    private Instant date;
```

Comment:
instead of private modifiers for class fields use annotation on class `@FieldDefaults(level = PRIVATE)`. Then remove all private modifiers for fields.